src/main/java/uk/gov/hmcts/ccd/domain/model/search/CriteriaInput.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.ccd.domain.model.search;
+
+public class CriteriaInput {
```

Comment:
could we rename this to SearchInput, and the CriteriaType to SearchType?
CriteriaInput doesn't convey the fact that these are classes used in the search, I find it a bit confusing name

Reply:
We have a binary naming of Search and workbasket. Hence naming with Search will be more confusing as if names with a word Search are related to the Search of search/workbasket. However, the concept of criteria is common to both Search and Workbasket, and when we need a seperation between a criteria on search or criteria on workbasket, the distinction is the context of the criteria. So, I think CriteriaType is better than SearchType. If we chose SearchType, there will be Search and Workbasket in it, causing more confusion for later, and particularly the newer readers.

Reply:
I think the 2 subclasses can also be removed. And we just leave SearchInupt